src/ReactLeafletMap/layers/CanvasVectorLayer/genericCanvasLayer.js

```diff
@@ -61,9 +61,11 @@ L.CanvasLayer = (L.Layer ? L.Layer : L.Class).extend({
 		}
 	},
 	_onLayerDidZoom: function (e) {
-		var topLeft = this._map.containerPointToLayerPoint([0, 0]);
-		L.DomUtil.setPosition(this._canvas, topLeft);
-		this.drawLayer();
+		if (this.props.type !== 'tiledVector') {
```

Comment:
What about move tiledVector type to core? We use it also in state. It is just suggestion

Reply:
or maybe we should move ale data source types to ptr-core...